tests/integration/components/identity-steps/step-one-test.js

```diff
@@ -0,0 +1,31 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'website-www/tests/helpers';
+import { render, click } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | identity-steps/step-one', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function (assert) {
+    this.set('startHandler', async () => {
```

Comment:
There's no assertion for this function, how we can be sure that this function worked?

Reply:
For now I have remove click handler as this test will cover on stepper-signup test to test the whole integration like after click on button what step should I have to show